patch/patch-runelite/src/main/kotlin/net/rsprox/patch/runelite/ProcessManager.kt

```diff
@@ -0,0 +1,62 @@
+package net.rsprox.patch.runelite
+
+public object ProcessManager {
+    private val processNamePattern = Regex("""\.runelite/repository2/client-.*?-(\d+)-patched\.jar""")
+
+    public fun findNewlyCreatedProcess(afterTimestamp: Long): String? {
+        val os = System.getProperty("os.name").lowercase()
+        try {
+            val processList = if (os.contains("win")) {
+                getWindowsProcesses()
+            } else {
+                getUnixProcesses()
+            }
+
+            processList.forEach { line ->
+                val matchResult = processNamePattern.find(line)
+                if (matchResult != null) {
+                    val matchedTimestamp = matchResult.groupValues[1].toLong()
+                    if (matchedTimestamp >= afterTimestamp) {
+                        return extractPid(line, os)
+                    }
+                }
+            }
+        } catch (e: Exception) {
+            e.printStackTrace()
```

Comment:
Use a logger instead of prints here.

Reply:
Haven't seen any examples of this in the codebase yet (all does this), but I pressume `logger.error { e.stackTraceToString() }` will do?

Reply:
Nah, logger.error(e) { "Message" }

It'll end up logging as:
ExceptionClass: Message
Stacktrace